inst/utils-0.0.1/addWidgetControls.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Adds a dropdown to highlight the selected group ID or set of group IDs in the chart.
+ *
+ * @param {Node} el - widget container, an element in the DOM
+ * @param {Array} dfResults - analysis results for a single metric
+ * @param {Object} lMetrics - metric metadata
+ * @param {Array} dfGroups - group metadata
+ *
+ * @returns {Node} HTML select element
+ */
+const addWidgetControls = function(el, dfResults, lMetrics, dfGroups, bAddGroupSelect) {
+    if (bAddGroupSelect === false)
```

Comment:
```suggestion
    if (!bAddGroupSelect)
```
I *think* we want to return the version without a control when bAddGroupSelect is malformed, too (not just when it's exactly `false`).